docs/user_guide/ug_general_concepts.md

```diff
@@ -2,40 +2,97 @@
 
 MechaPhlowers is a library specialized to perform mechanical and geometrical calculus for powerlines. The models are designed to be the simplest and most accurate possible : when possible, calculus are performed in 2D planes.
 
-The granularity is important : the line is composed by several section. For each section, you have several cables. Each cable can be divided into spans, which is the lowest scale of calculus.
+The granularity is important : the line is composed by several section. For each section, there are several cables. Each cable can be divided into spans, which is the lowest scale of calculus.
 
-[scheme granularity]
-Câble : Une phase peut être composée d’un unique câble ou d’un faisceau de câbles. Dans la suite on parle de câble de manière générique : phase constituée d'un câble simple, phase constituée d'un faisceau de câble, ou câble de garde.
-
-The algorithms are switching between the different levels during the resolution
+The algorithms are switching between the different levels during the resolution.
 
 ## Definition
 
 ![Image not available](./assets/powerline_definitions.drawio.png "Powerlines definitions")
 
-As you can see below, the study cable is hanging to different suspension strings and start/stopping to a dead end string.
+As described below, the study cable is hanging to different suspension strings and start/stopping to a tension support.
```

Comment:
As shown in the figure, the cable under study is suspended from suspension insulators and anchored to dead-end insulators.